### DISCUSSION TITLE
[operator] Can I pass a GitHub API Token to the trivy scanner Pods?

### DISCUSSION THREAD

**Discussion by consideRatio** [aquasecurity/starboard#210] [Questions & Help] [answered]
Can I configure the pods created by starboard-operator, or specifically, can I provide them with a GitHub token for Trivy to use in order to avoid being rate limited by GitHub? (https://github.com/aquasecurity/trivy#github-rate-limiting)

**Comment by danielpacak** [comment, marked as answer]
You cannot do that in v0.6.0 I opened https://github.com/aquasecurity/starboard/issues/212 to configure scan jobs created by the operator. In particular you'll be able to set a GitHub token for Trivy.
